Jenkinsfile

```diff
@@ -945,6 +945,33 @@ pipeline {
                         }
                     }
                 } // stage('Unit test Bullseye on EL 8')
+                stage('Unit Test Bullseye bdev on EL 8') {
+                    when {
+                        beforeAgent true
+                        expression { !skipStage() }
+                    }
+                    agent {
+                        label params.CI_UNIT_VM1_NVME_LABEL
+                    }
+                    steps {
+                        job_step_update(
+                            unitTest(timeout_time: 60,
+                                     unstash_opt: true,
+                                     ignore_failure: true,
+                                     inst_repos: prRepos(),
+                                     inst_rpms: unitPackages()))
+                    }
+                    post {
+                        always {
+                            // This is only set while dealing with issues
+                            // caused by code coverage instrumentation affecting
+                            // test results, and while code coverage is being
+                            // added.
+                            unitTestPostEx(ignore_failure: true,
+                                           ['covc_test_bdev_logs/', 'covc_vm_test/**'])
```

Comment:
Other than this log location, how is this stage any different than the one above/before it?  IOW, why doesn't the regular Bullseye stage test with `bdev` and what makes this one test with `bdev` given that there are no other changes to the stage (other than `bdev` in the name)?